app/containers/wallet/dialogs/WalletRestoreDialogContainer.js

```diff
@@ -51,6 +98,21 @@ export default class WalletRestoreDialogContainer extends Component<Props> {
       throw new Error('Unexpected restore mode: ' + this.props.mode);
     }
 
+    const { verifyRestore, submitValues } = this.state;
+    if (verifyRestore) {
+      const { addresses, accountPlate } = verifyRestore;
+      return (
+        <WalletRestoreVerifyDialog
+          addresses={addresses}
+          accountPlate={accountPlate}
+          onCopyAddress={() => {}}
```

Comment:
fake function should be fixed with a proper copy to clipboard

Reply:
The copy functionality works as it is. This function is only used as a callback on successful copying, and used only to display a notification. I have taken an example from other part of the code for paper-wallets where there's also this comment:

![image](https://user-images.githubusercontent.com/5585355/57587333-1bebb400-750c-11e9-9b19-10a21cfef39a.png)

I will put a comment for now here too, to avoid confusion. And maybe we want to create an issue to come up with a standard clipboard animation or something, and use it on all such cases.

Reply:
Or better yet, I will make this parameter optional for cases like this. Underlying component actually checks whether this callback is specified or not:

![image](https://user-images.githubusercontent.com/5585355/57587410-2ce8f500-750d-11e9-9f26-a5b4426975e3.png)